additional-functions/manage-restrictions.md

```diff
@@ -0,0 +1,49 @@
+## Manage restrictions
```

Comment:
Shouldn't this be rather part of RMS use case? Like an extension of below the standard use case.